js/solstice.cookieconsent.js

```diff
@@ -70,8 +70,7 @@ window.addEventListener("load", function() {
       }
     },
     onInitialise: function(status, options) {
-      var toolbar = document.getElementsByClassName("toolbar-manage-cookies");
-      if (toolbar.length <= 0) {
+      if (!document.getElementsByClassName("toolbar-manage-cookies").length) {
```

Comment:
I tested this locally and it does not work as is.

By doing this, it works:

```
if (!document.getElementsByClassName("toolbar-manage-cookies").length) {
  setTimeout(function() {
    document.getElementsByClassName("cc-revoke")[0].style.display = "block";
  });
}else {
  document.getElementsByClassName("cc-revoke")[0].style.display = "none";
}
```

Reply:
Thanks